pages/common/pyenv.md

```diff
@@ -0,0 +1,27 @@
+# pyenv  
+
+> Switch between multiple versions of Python easily.
+
+- List all available commands:
+
+`pyenv commands`
+
+- List all Python versions known to pyenv:
+
+`pyenv versions`
+
+- Install a Python version:
+
+`pyenv install {{version}}`
+
+- Uninstall a Python version:
+
+`pyenv uninstall {{version}}`
+
+- Set Python version to be used in global:
+
+`pyenv global {{version}}`
+
+- Set Python version to be used in local:
```

Comment:
Same: what does "local" mean exactly? With all the warning one sees around about the need for sandboxed python environments, it it hard to keep track of what meaning these terms have in the context of the different tools available to manage such environments.